cmd/create.go

```diff
@@ -96,14 +100,41 @@ func (cc *createCmd) initConfig() error {
 
 func (cc *createCmd) run() error {
 	log.Debugf("config: %s", cc.createConfigPath)
-	detectedLang, lowerLang, err := cc.detectLanguage()
+	var dryRunRecorder *dryrunpkg.DryRunRecorder
+	if dryRun {
```

Comment:
💡Question: I wonder from where this `dryRun` - value gets propagated?  I think at a `root` level like `root.go` file, please correct me.

Reply:
yes that's the way that it hands down the value, but we can remove the package-level var and instead parse the flag again if that would be clearer?

Reply:
i believe we use the same pattern for `provider` since it too belongs to the root command